RELEASING.md

```diff
@@ -56,3 +83,5 @@ Ensure you have a signed GPG key, and that the GPG key is listed in the Storm KE
 1. Go to http://repository.apache.org and drop the staging repository.
 
 2. Delete the staged distribution files from https://dist.apache.org/repos/dist/dev/storm/
+
+3. Delete the git tag.
```

Comment:
I don't think we should delete the git tag. Keeping it around makes it easier for people to find the source in git later, without having to go download the source zip (which would then not have any history). 

I think keeping it around is/has been standard procedure on this and other Apache projects

Reply:
This is about deleting the git tag when release candidate fails.  If it succeeds, we should keep the git tag.

Reply:
Ah, whoops.